lib/surgex/rpc/payload/request_payload.ex

```diff
@@ -2,7 +2,7 @@ defmodule Surgex.RPC.RequestPayload do
   @moduledoc false
 
   def encode(service_name, request_buf) do
-    Poison.encode!(%{
+    Jason.encode!(%{
```

Comment:
If you're removing devise_session, you're either way doing a breaking change which will force me to update major version. Therefore why wouldn't you remove RPC as well for which we have created a separate protein package as well?

In other words, either remove devise_session and rpc or leave both.